src/index.css

```diff
@@ -296,3 +302,70 @@ div#picker {
     width: 100%;
   }
 }
+.logsignin{
+  display: flex;
+  width: 100%;
+  justify-content: center;
+}
+button {
```

Comment:
Avoid giving the pure html elements their property, instead be specific to a class or id as custom components would also get affected (as they render the pure elements themselves ) and you might see abnormal behaviours later. Create a class instead and give class name to those elements like input and button.